geomstats/geometry/spd_matrices.py

```diff
@@ -21,19 +22,12 @@ def __init__(self, n):
             embedding_manifold=GeneralLinear(n=n))
         self.n = n
 
-    def belongs(self, mat, tolerance=TOLERANCE):
-        """Check if a matrix belongs to the manifold of SPD matrices."""
-        mat = gs.to_ndarray(mat, to_ndim=3)
-        n_mats, mat_dim, _ = mat.shape
-
-        mask_is_symmetric = self.embedding_manifold.is_symmetric(
-                mat, tolerance=tolerance)
-        mask_is_invertible = self.embedding_manifold.belongs(mat)
-
-        belongs = mask_is_symmetric & mask_is_invertible
-        belongs = gs.to_ndarray(belongs, to_ndim=1)
-        belongs = gs.to_ndarray(belongs, to_ndim=2, axis=1)
-        return belongs
+    def belongs(self, mat, atol=TOLERANCE):
+        """
+        Check if a matrix belongs to the manifold of
+        symmetric positive definite matrices.
+        """
+        return Matrices.is_symmetric(mat)
```

Comment:
yayy, this simplifies a lot indeed 🙏 !

Reply:
Shouldn't this method also check whether mat is invertible?

Reply:
Yes! had noticed as well 
This will be fixed by calling `GL.belongs` from the next PR:
```
GL.belongs(mat) = gs.where(gs.linalg.det(mat) != 0, True, false) 
```